examples/measure.js

```diff
@@ -0,0 +1,130 @@
+goog.provide('measure');
+
+goog.require('ngeo.DecorateInteraction');
+goog.require('ngeo.btngroupDirective');
+goog.require('ngeo.interaction.MeasureArea');
+goog.require('ngeo.interaction.MeasureAzimut');
+goog.require('ngeo.interaction.MeasureLength');
+goog.require('ngeo.mapDirective');
+goog.require('ol.FeatureOverlay');
+goog.require('ol.Map');
+goog.require('ol.Observable');
+goog.require('ol.Overlay');
+goog.require('ol.View');
+goog.require('ol.layer.Tile');
+goog.require('ol.source.OSM');
+
+
+/** @const **/
+var app = {};
+
+
+/** @type {!angular.Module} **/
+app.module = angular.module('app', ['ngeo']);
+
+
+/**
+ * App-specific directive wrapping the measure tools. The directive's
+ * controller has a property "map" including a reference to the OpenLayers
+ * map.
+ *
+ * @return {angular.Directive} The directive specs.
+ */
+app.measureToolsDirective = function() {
+  return {
+    restrict: 'A',
+    scope: {
+      'map': '=appMeasureToolsMap'
+    },
+    controller: 'AppMeasureToolsController',
+    controllerAs: 'measureCtrl',
+    bindToController: true,
+    templateUrl: 'partials/measuretools.html'
+  };
+};
+
+app.module.directive('appMeasureTools', app.measureToolsDirective);
+
+
+
+/**
+ * @param {ngeo.DecorateInteraction} ngeoDecorateInteraction Decorate
+ *     interaction service.
+ * @constructor
+ * @ngInject
+ */
+app.MeasureToolsController = function(ngeoDecorateInteraction) {
+
+  var style = new ol.style.Style({
+    fill: new ol.style.Fill({
+      color: 'rgba(255, 255, 255, 0.2)'
+    }),
+    stroke: new ol.style.Stroke({
+      color: 'rgba(0, 0, 0, 0.5)',
+      lineDash: [10, 10],
+      width: 2
+    }),
+    image: new ol.style.Circle({
+      radius: 5,
+      stroke: new ol.style.Stroke({
+        color: 'rgba(0, 0, 0, 0.7)'
+      }),
+      fill: new ol.style.Fill({
+        color: 'rgba(255, 255, 255, 0.2)'
+      })
+    })
+  });
+
+  /** @type {ngeo.interaction.MeasureLength} */
+  var measureLength = new ngeo.interaction.MeasureLength({
+    sketchStyle: style
+  });
+  measureLength.setActive(false);
+  ngeoDecorateInteraction(measureLength);
+  this['map'].addInteraction(measureLength);
+  this['measureLength'] = measureLength;
+
+  var measureArea = new ngeo.interaction.MeasureArea({
+    sketchStyle: style
+  });
+  measureArea.setActive(false);
+  ngeoDecorateInteraction(measureArea);
+  this['map'].addInteraction(measureArea);
```

Comment:
I'd create a local `map` variable instead of repeating `this['map']`. That's minor.